test/models/voice_response_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+class VoiceResponseTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+  
+  test 'voice response accepts BeginCall object' do
+    call = BeginCall.new
+    response = VoiceResponse.new(call)
+  end 
+  
+  test 'voice response accpets ForwardCall object' do
+    call = ForwardCall.new
+    response = VoiceResponse.new(call)
+  end 
+  
+  test 'voice response throws exception for different object' do
+    class DummyCall
+    end 
+    call = DummyCall.new
+    assert_raise RuntimeError do VoiceResponse.new(call) end
```

Comment:
I believe it's `assert_raises` not `assert_raise`. Is there any `NoMethodError`s thrown? If not, Rails might just be adding an alias, which would be fine as well.

For single line blocks, use curly braces, or put `do ... end` blocks on multiple lines.

```ruby
assert_raises RuntimeError { VoiceResponse.new(call) }
```

```ruby
assert_raises RuntimeError do
  VoiceResponse.new(call)
end
```